chia/farmer/farmer_api.py

```diff
@@ -27,6 +27,16 @@
 from chia.util.ints import uint32, uint64
 
 
+def strip_old_entries(pairs: Sequence[Tuple[float, Any]], before: float):
```

Comment:
I would add return type annotation for clarity

Reply:
Quite so, thanks for pointing it out.  Fixed.